ding/model/common/head.py

```diff
@@ -1079,7 +1079,7 @@ class ReparameterizationHead(nn.Module):
         ``__init__``, ``forward``.
     """
 
-    default_sigma_type = ['fixed', 'independent', 'conditioned']
+    default_sigma_type = ['fixed', 'independent', 'conditioned', 'happo']
```

Comment:
If this special type not being used in other algos, why not creating a new class for modeling policy of happo?